### DISCUSSION TITLE
Article 4 directions

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#30] [Planning consideration]
# Article 4 directions

An Article 4 direction is made by the local planning authority to remove all or some of the permitted development rights on a site in order to protect it

## What

We have created [a standard for Article 4 directions](https://digital-land.github.io/specification/specification/article-4-direction/) - this standard is currently in the piloting stage. Piloting means we have developed a standard, a handful of Local planning authorities are publishing data to the standard and the [planning data platform is collecting it](https://www.planning.data.gov.uk/dataset/article-4-direction-area).

## Why

Anyone looking to develop a site needs to know if a site has had development rights removed. This should be easy for computers to determine. However, there isn't a complete digital record of all Article 4 directions.

## What we're looking for

We'd like to know:

* if the standard covers everything about Article 4 directions you'd expect it to, and if not, what you think is missing
* are you using the data, if so, what for?
* what you think would improve the data?
* what are the difficulties, if any, in creating and publishing this data
* anything else you think we should know

**Comment by colmjude** [comment]
Some local planning authorities publish their article 4 direction data, including:

* [Newcastle](https://datamillnorth.org/dataset/newcastle-article-4-directions)

**Comment by mattlucht** [comment]
In a conversation with some LPAs they mentioned that there a multiple dates related to an Article 4. Namely, when the order was `made` and when it was `confirmed`. This is discussed in the [issue for TPOs](https://github.com/digital-land/data-standards-backlog/issues/11#issuecomment-1640069500)

**Reply by colmjude** [reply]
It would be good to know more about this.

Does the process for creating article-4-directions involve a period between when it is first "made" and then ultimately "confirmed"? 
And, if it does, is there a set time between the "made" and "confirmed" stages? We understand that for TPOs this gap is a maximum of 6 weeks.

If any LPAs follow this approach, could they share a few examples of these dates in relation to their existing article-4-directions? We can use that as evidence to add these fields to the specification if necessary.

**Reply by NoraAC** [reply]
hi, we have a very similar issue with 5 new article 4s to be issued in the next couple of weeks but they won't be adopted/take effect until after June 2026. Currently we are using 'start-date' to express when the requirements are taking effect, however in this situation we would have two overlapping start dates. We could keep the 'start-date' to refer to June 2026 or it might be clearer to have a separate 'effective data' which relates to the time when the regulations come into effect. 
This won't necessarily affect the data that has been added already, as when different systems interrogate it could check if there is an 'effective-date' , if yes show that, but if no 'effective-date' and no 'end-date' can show 'start-date'

**Reply by kieranwint** [reply]
Thanks for your insight @NoraAC, do you know if this is something that is quite widespread?

**Comment by mattlucht** [comment]
There are some changes that we're proposing to make to the [Article 4 Direction specification](https://digital-land.github.io/specification/specification/article-4-direction/). If/when we make these changes we'll ensure that any existing data is backward compatible with the new specification.

Any comments or feedback is much appreciated!

* we would like to change the reference dataset from [Article 4 Direction rule](https://www.planning.data.gov.uk/dataset/article-4-direction-rule) to a `permitted-development-right` dataset (at the moment we need to do a bit of work on the PDR dataset but that'll be there soon). Any existing records that refer to `article-4-direction-rule` we'll liaise with the relevant local planning authority and do a manual migration
* We'll remove `document-url` from `article-4-direction-area`
* We'll add `point` to `article-4-direction-area` (note: if an LPA has provided a `geometry` then we'll use that to determine a mid-point `point`)

**Reply by colmjude** [reply]
We've made these changes and bumped the [spec version to 1.2.2](https://digital-land.github.io/specification/specification/article-4-direction/)

**Comment by colmjude** [comment]
### Article 4 direction area types?

We have an example where a local planning authority has provided us 2 article-4-direction-areas. One is for a current area. One is for a proposed area.
We need to understand if `proposed` areas have any material impact on any planning process? This will help us decide if `proposed` areas need to be captured in the data

**Comment by colmjude** [comment]
## Update to the data model

We are going to make a small change to the `article-4-direction-area` dataset.

Currently, it has `uprn` and `address-text` fields. These fields expect a single value. A single uprn in `uprn` and a single address in `address-text`. However, our understanding is an article 4 direction can reference multiple specific properties. This means these fields need to accept multiple values.
We are planning to change them to

* `uprn` to `uprns`
* `address-text` to `address-texts`

**Reply by colmjude** [reply]
We have now made these changes.

See commits:
* [b939e11](https://github.com/digital-land/specification/commit/b939e11)
* [a3ceb2c](https://github.com/digital-land/specification/commit/a3ceb2c)
* [45abb1f](https://github.com/digital-land/specification/commit/45abb1f)

**Comment by kieranwint** [comment]
## OS Update

We have been doing some work with Ordnance Survey on sharing derived data under the open government licence, and the PSGA. 

As a result of this collaboration, they have recently updated one of their [webpages](https://www.ordnancesurvey.co.uk/customers/public-sector/public-sector-licensing/publish-derived-data), under the “planning” sub-heading, to state that “article 4 direction area polygons” fall under the presumption to publish criteria, based off of examples we gave (those being a large rural area and the cumulative extent of an estate).

If you have any further questions on this, please let us know!

**Comment by colmjude** [comment]
## Moving to candidate

We have been piloting [this specification](https://digital-land.github.io/specification/specification/article-4-direction/) for a while with a small number of data providers.

What we’ve learnt during this phase has helped us to tweak and improve the data specification so that it works for the community of data providers and consumers.

Analysis of the data produced to this specification so far suggests the specification is working for the vast majority of data providers.

Providers have been able to publish data for [article-4-direction](https://www.planning.data.gov.uk/dataset/article-4-direction) and [article-4-direction-area](https://www.planning.data.gov.uk/dataset/article-4-direction-area) datasets. In some cases, all the expected fields are not there but that's okay. We want providers to first share what they have or can easily get hold of. Then they can fill in the gaps afterwards. For example, more providers are starting to list the permitted development rights that are removed in the `permitted-development-right-field`.

We are going to move the article 4 direction specification from “pilot” specification to “candidate” specification.

This means we’re confident that the specifciation is fit for purpose, works for providers and produces valuable data.

The specification may still expand and evolve but we don’t expect to have to drastically overhaul it.

During the candidate stage, the specification will be tested with a larger group of data providers and reviewed by our legal teams before being worked into legislation as part of the [Levelling Up and Regeneration Act 2023](https://www.legislation.gov.uk/ukpga/2023/55/contents/enacted).

Thanks to everyone who has been involved in getting us to this point. We appreciate all your contributions.

Any questions, please do ask.

**Comment by kieranwint** [comment]
### Historic permitted development rights

We’ve recently been doing some research into historic [permitted development rights](https://github.com/digital-land/data-standards-backlog/discussions/39#discussioncomment-12867502) (PDRs), and we’re actively looking into how historic PDRs affect Article 4 Directions from both a legal and planning perspective, and we’ve asked policy colleagues for support on this.

If you have any thoughts on this, please do get involved.

**Comment by colmjude** [comment]
## Recording agreed Article 4 Directions before they’re active

When an LPA formally agrees an Article 4 Direction (A4D), it is helpful to publish the data immediately, even though it doesn’t take legal effect until a later date. Recording these A4Ds as soon as they’re agreed, but before they’re active, is helpful because:

- **Transparency for users** - Developers and planning officers can see what new restrictions are coming, even if they can’t be applied yet. This avoids surprises when the A4D finally takes effect.
- **Avoiding Compensation Risk**  - Once an A4D is agreed (but not yet effective), the LPA must provide notice so that, from the effective date onwards, properties won’t lose permitted development rights without fair warning. Recording the A4D early ensures the “waiting period” is visible.

Below is an example table showing how an agreed A4D would be captured in our dataset. The `entry-date` is when it’s published (agreed), and the `start-date` is when it becomes legally active (effective).

| reference   | name  | description | documentation-url | document-url | entry-date  | start-date | end-date |
|---|---|---|---|---|---|---|---|
| A4D-CMD-001 | Interim Article 4 Direction (Camden) | Restricts change-of-use of certain buildings | https://camden.gov.uk/planning/a4d-001/info     | https://camden.gov.uk/planning/a4d-001/doc.pdf  | 2025-06-09  | 2026-06-09  |  |

- **`entry-date` (2025-06-09):** Date the A4D is published (agreed).  
- **`start-date` (2026-06-09):** Future date when the A4D takes legal effect.  
- **`end-date`** remains blank until this A4D is superseded or expires.

By capturing both `entry-date` and `start-date`, we clearly show that the A4D is “on the shelf” (agreed but inactive) and track exactly when it becomes legally binding.

**Reply by NoraAC** [reply]
Great thx Colm. We are looking to add this information to land charges as well, and it seems that they only record when it was added in land charges as a registration, when it expires and when it is deactivated, so the name and notes would be the main fields to inform about the status of the Article4. We've decided to add in the name 'Non-immediate Article 4' instead of 'Interim' for more clarity. 

On another note, the Extract from Incubator for AI seems to record an effective date when it scans the Article 4 documents, which will then be shared with planning.data so I wonder how would that be mapped, following the above logic. thx. https://ai.gov.uk/projects/extract/
